docs/partials-common/install-espidf.md

```diff
@@ -26,10 +26,8 @@ sudo apt-get install git wget flex bison gperf python3 python3-venv \
 ```console
 mkdir -p ~/esp
 cd ~/esp
-git clone --recursive https://github.com/espressif/esp-idf.git
+git clone --recursive https://github.com/espressif/esp-idf.git -b v5.1.1
```

Comment:
Nice, much cleaner!

Reply:
This is the recommended ESP-IDF approach and I agree it feels cleaner.